packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -124,7 +132,8 @@ export const FeedContainer = ({
   const numCards = currentSettings.numCards[spaciness ?? 'eco'];
   const insaneMode = !forceCardMode && listMode;
   const isList = (insaneMode && numCards > 1) || shouldUseFeedLayoutV1;
-  const feedGapPx = getFeedGapPx[gapClass(isList, spaciness)];
+  const feedGapPx =
```

Comment:
If we already have a function that does business logic to assign a variable, I'd rather keep all the logic in the function. But that's my preference, not sure what is the convention in the code 🤷

Reply:
We don't have any specific convention for this one, just thought of not touching the function and having it lesser parameters than it needs to when we can make the decision outside. But yes, I also agree that since it already handles the business logic, why not have it inside? I am cool with what you have here, I just threw some thoughts.